app/helpers/service_helper.rb

```diff
@@ -0,0 +1,50 @@
+module ServiceHelper
+  private
+
+  def custom_attr_fields(host, custom_attrs)
+    validated_fields = []
+    custom_attrs.each do |attr_name, fields|
+      fields = downcase_keys(fields)
+      value = fields['value'].to_s
+      fields['value'] = filter_vulnerabilities(host, attr_name, value)
+      validated_fields << fields
+    end
+    validated_fields
+  end
+
+  def downcase_keys(raw_hash)
+    hash_with_downcase_key = {}
+    raw_hash.each_pair { |k, v| hash_with_downcase_key[k.downcase] = v }
+    hash_with_downcase_key
+  end
+
+  # rubocop:disable Security/Eval, Style/StringLiterals, Lint/UnusedMethodArgument
+  def filter_vulnerabilities(host, attr_name, value)
```

Comment:
I don't want this extra overhead. The person making the config file has root access to the machine running the code and can be trusted.

Reply:
Ok. I will do so